service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/AiNotebookApiConversions.java

```diff
@@ -0,0 +1,32 @@
+package bio.terra.workspace.service.resource.controlled.cloud.gcp.ainotebook;
+
+import bio.terra.workspace.generated.model.ApiGcpAiNotebookInstanceAcceleratorConfig;
+import com.google.api.services.notebooks.v1.model.AcceleratorConfig;
+import javax.annotation.Nullable;
+
+/** Utility method for converting AI notebook objects between WSM formats and GCP formats. */
+public class AiNotebookApiConversions {
```

Comment:
If you make a record or class for the internal and serializable format of the accelerator config, you can put these methods into that class.